src/ng2-smart-table/components/cell/edit-cell.component.ts

```diff
@@ -10,9 +10,31 @@ export class EditCellComponent {
 
   @Input() cell: Cell;
   @Input() inputClass: string = '';
+  @ViewChild('dynamicTarget', {read: ViewContainerRef}) dynamicTarget: any;
+  customComponent: any;
 
   @Output() public edited: EventEmitter<any> = new EventEmitter<any>();
 
+  constructor(private resolver: ComponentFactoryResolver){
+  }
+
+  ngOnChanges(arg) {
```

Comment:
> This also should be moved into the separate CustomEditorComponent so that we don't have this logic here

Would you suggest to create a separate class with the _custom editor_ methods and extend `edit-cell-component` to clean things up or to create 2 separate independent components to handle the _custom editor_ and the _text editor_?